xml/signals/Reading-1954/appearance-SL-2-high-abs.xml

```diff
@@ -38,60 +38,43 @@
 
   <appearances>
 
-    <appearance>
-      <aspectname>Clear alt</aspectname>
-      <show>green</show>
-      <show>green</show>
-      <reference>Page 92</reference>
-      <comment>Unofficial Rule and Aspect Name to allow for optional appearance</comment>
-      <imagelink>../../../resources/icons/smallschematics/aspects/Reading-1954/SL-2-high-abs/rule-281-alt.gif</imagelink>
-    </appearance>
-
     <appearance>
       <aspectname>Clear</aspectname>
       <show>green</show>
       <show>red</show>
-      <reference>Page 92</reference>
+      <reference>Page 76</reference>
       <imagelink>../../../resources/icons/smallschematics/aspects/Reading-1954/SL-2-high-abs/rule-281.gif</imagelink>
     </appearance>
 
     <appearance>
       <aspectname>Approach Medium</aspectname>
       <show>yellow</show>
       <show>green</show>
-      <reference>Page 94</reference>
+      <reference>Page 79</reference>
       <imagelink>../../../resources/icons/smallschematics/aspects/Reading-1954/SL-2-high-abs/rule-282.gif</imagelink>
     </appearance>
 
     <appearance>
       <aspectname>Advance Approach</aspectname>
       <show>yellow</show>
       <show>yellow</show>
-      <reference>Page 95</reference>
+      <reference>Page 80</reference>
       <imagelink>../../../resources/icons/smallschematics/aspects/Reading-1954/SL-2-high-abs/rule-282A.gif</imagelink>
     </appearance>
 
     <appearance>
       <aspectname>Approach</aspectname>
       <show>yellow</show>
       <show>red</show>
-      <reference>Page 98</reference>
+      <reference>Page 84</reference>
       <imagelink>../../../resources/icons/smallschematics/aspects/Reading-1954/SL-2-high-abs/rule-285.gif</imagelink>
     </appearance>
 
-    <appearance>
-      <aspectname>Restricting</aspectname>
-      <show>red</show>
-      <show>yellow</show>
-      <reference>Page 103</reference>
-      <imagelink>../../../resources/icons/smallschematics/aspects/Reading-1954/SL-2-high-abs/rule-290.gif</imagelink>
-    </appearance>
```

Comment:
Could it cause problems for users if this is removed?

Reply:
Depends on whether the 2-head, high, absolute mast _has_ a Restricting indication available _in the applicable rulebook_.

When I reference https://www.railroadsignals.us/rulebooks/rdg54/index.htm , I note that a signal of the form shown in Rule 281 illustration "C" does _not_ appear in the "Rule 290 Restricting" portion of the table.  Based on the _assumption_ that the Reading "2-head high, absolute" mast is of the form shown in 281-C, it ought to be safe.

But your earlier comment about 1956 versus 1954 makes this questionable.  Someone with the 1956 rulebook will need to weigh in on this part of the question.